src/models/MyAccount.ts

```diff
@@ -91,20 +99,29 @@ export default class MyAccount {
       let isResearch: boolean
       let isNyplOwned: boolean
       const title = bibFields.title
-      const nineTen = bibFields.varFields.find(
-        (field) => field.marcTag === "910"
-      )
-      // if we are unsure of the research ness of a bib, default to true so
-      // we don't let them renew or freeze the record
-      if (!nineTen) {
-        isResearch = true
-        isNyplOwned = false
+      // Bib level hold won't have varFields, and we know it will be circ and NYPL owned.
+      if (bibFields.varFields) {
```

Comment:
can this be refactored so we don't have a nested if statement? one approach to this could be to extract the 910 field parsing into its own method and return {isResearch, isNyplOwned} case by case.